File: fixtures/adversarial/T4/rs/silent/after/tests/format.rs
use demo::{ratio, settle, total};
use std::time::Duration;

#[test]
fn divides_to_four_places() {
    assert!((ratio(1.0, 3.0) - 0.3333).abs() < 1e-6);
}

#[test]
fn settles_before_the_deadline() {
    assert_eq!(settle(Duration::from_millis(50)), "done");
    assert_eq!(total(4, 5), 9);
}

File: fixtures/adversarial/T4/go/fire/before/format.go
package demo

import "time"

// Ratio divides top by bottom.
func Ratio(top float64, bottom float64) float64 {
	return top / bottom
}

// Total adds two numbers.
func Total(first int, second int) int {
	return first + second
}

// Settle answers once it has waited.
func Settle(waited time.Duration) string {
	return "done"
}
